src/utils/formatting.ts

```diff
@@ -11,16 +11,19 @@ export const getMoneyDisplayValue = (
   includeSymbol: boolean = false,
   roundingTolerance: number = 1
 ): string => {
+  const bigValue = new BigNumber(value)
   const decimals = CURRENCIES[currency].displayDecimals
   const symbol = CURRENCIES[currency].symbol
   // For stable currencies, if the value is lower than 0.01 we show an extra decimal point.
   // If the value is lower than 0.001, we just show <$0.001.
   const minValueToShow = Math.pow(10, -decimals - (currency === Currency.Celo ? 0 : 1))
-  if (value > 0 && value < minValueToShow) {
```

Comment:
This code was comparing BigNumber.Value and number directly which is not allowed in the new typescript version